### ISSUE TITLE
Screen Reader is not announcing the status messages that appeared on the screen.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13997]
**Test Environment**
OS: Windows_11
Version: 21H2
OS Build: 22000.795
Browser: Version 103.0.1264.71 (Official Build) (64-bit)
URL: https://clickabledemo.ppe.experience.retail.ms/workspace/hybrid-workplace/category/en-US/2f0975b6-2a02-4f3a-ff54-1a520aac9531

**Repro Steps:**
1. Open the above URL in Edge browser.   
2. Navigate to the Backup files in OneDrive.
3. Click on Start tutorial, welcome demo page will be appeared.
4. Verify whether Screen Reader is announcing the status message or not in welcome demo page.

 **Actual Behavior**
Screen Reader is not announcing the status messages that appeared on the screen.

**Expected Behavior**
Screen reader should announce the status messages that appeared on the screen.
Screen reader should announce as 'Welcome to the demo: Back up files in one drive'.

https://user-images.githubusercontent.com/93735775/183417096-15c10b4a-3f01-408a-84cf-064d70710987.mp4

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- Please fill out the issue template fully.
- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

Kind Regards,
NV Access Software Developers
